app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -1045,6 +1069,25 @@ public void setBlockingEnabled(boolean enabled) {
         }
     }
 
+    private void showSecurityPopUp() {
+        final PopupWindow securityPopup = PopupUtils.createSecurityPopup(getContext(), session);
+        if (securityPopup != null) {
+            securityPopup.setOnDismissListener(new PopupWindow.OnDismissListener() {
+                @Override
+                public void onDismiss() {
+                    popupTint.setVisibility(View.GONE);
+                }
+            });
+            securityPopup.setAnimationStyle(android.R.style.Animation_Dialog);
+            popupTint.setVisibility(View.VISIBLE);
```

Comment:
over-nitpicking. You have a bunch of calls on "securityPopup" here and in between a call on popupTint. To make this more readable I'd move this line to the end.